src/areaSearch/areaSearchSpecsPage.tsx

```diff
@@ -157,10 +156,12 @@ const AreaSearchSpecsPage = ({
     if (!prevSubmissionIdRef.current) {
       change(AREA_SEARCH_FORM_NAME, 'form', applicationFormTemplate, true);
     }
+  }, []);
+
+  useEffect(() => {
     if (lastSubmissionId > prevSubmissionIdRef.current) {
       prevSubmissionIdRef.current = lastSubmissionId;
       setSubmitSucceeded(AREA_SEARCH_FORM_NAME);
```

Comment:
Would it make sense to listen to the action `RECEIVE_AREA_SEARCH_SAVED` and then trigger a saga that does this set submit succeeded?

Reply:
I will try

Reply:
So you think that it makes sense? It was a question :)

Reply:
I looked at it and it does not really make sense.

Reply:
The saga module should be connected to the redux form state in order to be able to handle it and I am not really sure if it even works. The way it is implemented now is more lean and less gimmicky.